### PR TITLE
BRO remaining Prototype cards

### PR DESCRIPTION
**Pull request by Simisays**
Coudn't get Rootwire Amalgam to work without the StoreSVar

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/depth_charge_colossus.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Name:Depth Charge Colossus
+ManaCost:9
+Types:Artifact Creature Dreadnought
+PT:9/9
+K:Prototype:4 U U:2:2
+K:CARDNAME doesn't untap during your untap step.
+A:AB$ Untap | Cost$ 3 | SpellDescription$ Untap CARDNAME
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/depth_charge_colossus.txt:7]: add `.`

File: forge-gui/res/cardsfolder/upcoming/rootwire_amalgam.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Name:Rootwire Amalgam
+ManaCost:5
+Types:Artifact Creature Golem
+PT:5/5
+K:Prototype:1 G:2:3
+A:AB$ Token | Cost$ 3 G G Sac<1/CARDNAME> | TokenAmount$ 1 | TokenScript$ c_x_x_a_golem | TokenPower$ X | TokenToughness$ X | PumpDuration$ EOT | TokenOwner$ You | PumpKeywords$ Haste | SorcerySpeed$ True | SubAbility$ StorePower | SpellDescription$ Create an X/X colorless Golem artifact creature token, where X is three times Rootwire Amalgam's power. It gains haste until end of turn. Activate only as a sorcery.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/rootwire_amalgam.txt:6]: CARDNAME

File: forge-gui/res/cardsfolder/upcoming/skitterbeam_battalion.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Name:Skitterbeam Battalion
+ManaCost:9
+Types:Artifact Creature Construct
+PT:4/4
+K:Trample
+K:Haste
+K:Prototype:3 R R:2:2
+T:Mode$ ChangesZone | Destination$ Battlefield | ValidCard$ Card.Self+wasCast | Execute$ TrigToken | TriggerZones$ Battlefield | TriggerDescription$ When CARDNAME enters the battlefield, if you cast it, create two tokens that are copies of it.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/skitterbeam_battalion.txt:8]: wasCastByYou

File: forge-gui/res/cardsfolder/upcoming/skitterbeam_battalion.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+SVar:TrigToken:DB$ CopyPermanent | Defined$ TriggeredCard  | NumCopies$ 2
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/skitterbeam_battalion.txt:9]: spacing

File: forge-gui/res/cardsfolder/upcoming/iron_craw_crusher.txt [hunk 5]
```diff
@@ -0,0 +1,11 @@
+Name:Iron-Craw Crusher
+ManaCost:7
+Types:Artifact Creature Wurm
+PT:4/6
+K:Prototype:2 G G:2:5
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigPump | TriggerDescription$ Whenever CARDNAME Crusher attacks, target attacking creature gets +X/+0 until end of turn, where X is CARDNAME's power.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/iron_craw_crusher.txt:6]: check TrigDesc

File: forge-gui/res/cardsfolder/upcoming/iron_craw_crusher.txt [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+SVar:TrigPump:DB$ Pump | ValidTgts$ Creature.attacking | TgtPrompt$ Select target attacking creature you control | NumAtt$ X
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/iron_craw_crusher.txt:7]: no "you control" unless the oracle here is wrong? Either way something needs to be fixed

File: forge-gui/res/cardsfolder/upcoming/rootwire_amalgam.txt [hunk 7]
```diff
@@ -0,0 +1,11 @@
+Name:Rootwire Amalgam
+ManaCost:5
+Types:Artifact Creature Golem
+PT:5/5
+K:Prototype:1 G:2:3
+A:AB$ Token | Cost$ 3 G G Sac<1/CARDNAME> | TokenAmount$ 1 | TokenScript$ c_x_x_a_golem | TokenPower$ X | TokenToughness$ X | PumpDuration$ EOT | TokenOwner$ You | PumpKeywords$ Haste | SorcerySpeed$ True | SubAbility$ StorePower | SpellDescription$ Create an X/X colorless Golem artifact creature token, where X is three times CARDNAME's power. It gains haste until end of turn. Activate only as a sorcery.
+SVar:StorePower:DB$ StoreSVar | SVar$ X | Type$ Number | Expression$ 1
```
[the remark this answers is not in the document]
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/rootwire_amalgam.txt:7]: The token died at the beginning of my next turn. Gonna test some more

**Review by tool4ever — APPROVED** [review]

**Pull request merged**